### DISCUSSION TITLE
Survey Callback

### DISCUSSION THREAD

**Discussion by Schypozhoa** [LiveHelperChat/livehelperchat#1967] [Q&A] [answered]
Hello,

I'm using LHC v4.09
I want to redirect my user to another page after they complete the survey, Is there any way to detect if user is complete filling the survey in chat widget?

Thanks

**Comment by remdex** [comment, marked as answer]
At the moment there is *no* JS callback for that event. But you can just override template `design/defaulttheme/tpl/lhsurvey/fillwidget.tpl.php` and add JS there like `window.parent....`

**Reply by Schypozhoa** [reply]
> At the moment there is JS callback for that event 

What is the JS callback for that event? I didn't find anything about surveys in the JS callback documentation (or I missed it?). I want to use it in JS along with other callback like 'showWidget'.

**Reply by remdex** [reply]
There is no js callback, mistyped. So you will have to override template just
